frontend/src/components/Shared/HeroBillboard/HeroBillboard.jsx

```
@@ -1,101 +1,103 @@
+          src={getImageUrl()} 
           alt={content.title} 
-          className="w-full h-full object-cover opacity-70" 
+          className="w-full h-full object-cover opacity-70"
+          onError={(e) => {
+            e.target.src = `/api/placeholder/1200/600?text=${encodeURIComponent(content.title)}`;
+          }}
         />
       </div>
-      
-      {/* Gradient overlay */}
       <div className="absolute inset-0 bg-gradient-to-t from-gray-900 via-transparent to-transparent z-1"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)